app/src/main/java/com/jetbrains/handson/mpp/mobile/CustomListAdapter.kt

```diff
@@ -0,0 +1,49 @@
+package com.jetbrains.handson.mpp.mobile
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import java.text.SimpleDateFormat
+import java.util.*
+
+class CustomListAdapter(private val tickets: List<TicketInfo>, private val context: Context) : RecyclerView.Adapter<CustomListAdapter.CustomViewHolder>() {
+    class CustomViewHolder(private val view: View, private val context: Context): RecyclerView.ViewHolder(view) {
+        fun bindItems(departureTime: String, arrivalTime: String, maxPrice: Int, minPrice: Int) {
+            val colDepTime = view.findViewById<TextView>(R.id.departure_time)
```

Comment:
Originally "column", although when I named these variables I hadn't fully understood how this adapter was working so carried over a naming habit from a previous UI project I did (which doesn't make sense now I understand how this adapter works). I have updated to `depTime`, `arrTime` and `prices` to avoid confusion.